src/cpp/desktop/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ if(NOT WIN32)
          endif()
       endif()
 
-      set(QT_CANDIDATES 5.12.8)
+      set(QT_CANDIDATES 5.12.8 5.12.10)
```

Comment:
Note: made this change just so development builds can continue to work (since 5.12.8 isn't made available by the online installer, but 5.12.10 is)

Reply:
5.12.8 is still available in the online installer. Just click the LTS checkbox and Filter, and all versions reappear.